docs/detections/alerts-view-details.asciidoc

```diff
@@ -22,29 +22,54 @@ image::images/alert-details-flyout.png[Alert details flyout]
+* *Highlighted fields*: Surfaces the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
 
-* *Highlighted fields*: The most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
-+
-NOTE: The *Session ID* field provides a unique ID for tracking a given Linux session and is stored in the `process.entry_leader.entity_id` field in the alert's document. To collect the session ID and other session data, you must enable the *Include session data* setting on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
-
-* *Alert prevalence*: The total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for `host.name` means three alerts with the same `host.name` value exist within the timeframe.
+* *Alert prevalence*: Shows the total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for `host.name` means three alerts with the same `host.name` value exist within the timeframe.
 +
 Alert prevalence data can help you investigate relationships with other alerts and gain context about the events producing alerts. You can also click the alert prevalence count to explore the alerts in Timeline.
 +
 IMPORTANT: Before investigating alert prevalence data in Timeline, save any Timelines you're working on to ensure you can access them later.
 
-* *Enriched data*: Available threat indicator matches and threat intelligence data. This section only displays for alerts with intelligence data. Click the info icon to learn more about what data is collected. Refer to <<enriched-data-overview>> for more information.
+* *Insights*: Shows relationships with related alerts to help you quickly identify patterns. Refer to <<alert-details-insights, Insights on alerts>> for more information.
+
+* *Enriched data*: Displays available threat indicator matches and threat intelligence data. This section only displays for alerts with intelligence data. Click the info icon to learn more about what data is collected. Refer to <<enriched-data-overview>> for more information.
 +
 [role="screenshot"]
 image::images/enriched-data-info-icon.png[Informational message on enriched data, 600]
 
+[discrete]
+[[alert-details-insights]]
+==== Insights on alerts
+
+The Insights section shows you how an alert is related to other alerts and offers ways to investigate related alerts. You can use this information to quickly find patterns between alerts and then take action.
+
+Within the Insights section, you can click on the title for each insight to expand or collapse it. Some sections are only offered under a certain license and might require you to also enable a feature flag.
```

Comment:
Licensing and feature flags are mentioned in two places in this section, first here where it all sounds kind of intentionally vague, then later when we actually reveal all the details. I think we could omit the second sentence here, and just mention licensing and feature flags below, where it's directly related to the features and details.
```suggestion
Within the Insights section, you can click on the title for each insight to expand or collapse it.
```

Reply:
Sounds good. I wasn't sure whether this was something worth mentioning early, especially considering the image below it shows the insights section that needs the plat license and feature flag enabled.

Reply:
I think if it was something we could fully explain pretty quickly upfront, then yeah. But it's kind of detailed and would bog things down if we included all of the details upfront.